include/esp_littlefs.h

```diff
@@ -19,6 +19,11 @@ extern "C" {
 #define ESP_LITTLEFS_VERSION_MINOR 16
 #define ESP_LITTLEFS_VERSION_PATCH 2
 
+#ifdef ESP8266
+// ESP8266 RTOS SDK default enables VFS DIR support
+#define CONFIG_VFS_SUPPORT_DIR 1
+#endif
```

Comment:
This define here seems inappropriate here; does `VFS_SUPPORT_DIR` not exist in es8266 rtos sdk? Would it be better to define it in CMakeLists?

Reply:
The ESP8266 RTOS SDK feels like a "degraded" version of ESP-IDF 📦 
It didn't have configurable dir support, it just have them as a part of standard APIs: https://github.com/espressif/ESP8266_RTOS_SDK/blob/master/components/vfs/include/esp_vfs.h

Consider most of the official dev resources have shifted to ESP32, I suspect this SDK will stay in this way for a long time, so this seems to be a reasonable "patch" to make things just work.

Fingers crossed if Espressif decided to give the ESP8266 SDK another refresh (to be more aligned with later version of ESP-IDF), this quick and dirty snippet can be just removed.

WDYT?

Reply:
seems reasonable to me 👍